src/apps/opening-hours/OpeningHourWeekList.tsx

```diff
@@ -0,0 +1,48 @@
+import dayjs from "dayjs";
+import "dayjs/locale/da";
+import React from "react";
+import OpeningHoursDayEntry from "./OpeningHoursDayEntry";
+import { GroupedOpeningHours } from "./OpeningHoursHelpers";
+import OpeningHourWeekListSkeleton from "./OpeningHourWeekListSkeleton";
+
+interface OpeningHoursWeekListProps {
+  data: GroupedOpeningHours;
+  isLoading: boolean;
+}
+
+const OpeningHoursWeekList: React.FC<OpeningHoursWeekListProps> = ({
+  data,
+  isLoading
+}) => {
+  return isLoading ? (
+    <OpeningHourWeekListSkeleton />
+  ) : (
+    <ul className="opening-hours__content">
+      {data.map(({ day, date, openingHourEntries }) => {
+        const formattedDateForDisplay = dayjs(date).format("DD/MM");
+        return (
+          <li key={date} className="opening-hours__row">
+            <h3 className="opening-hours__individual-day">{`${day}: d. ${formattedDateForDisplay}`}</h3>
```

Comment:
Please ensure that our translation handling is used for this text.

Reply:
Added